masterlist.yaml

```
@@ -3544,7 +3544,13 @@ plugins:
         condition: 'not (file("Particle Patch for ENB.bsa") and active("Particle Patch for ENB.esp")) and not file("textures/effects/gradients/gradillusiondark01m.dds") and not file("textures/WiZkiD/iron_e.dds")'
     tag: [ Graphics ]
 
-  - name: 'ImprovedHearthfireLighting.esp'
+- name: 'ImprovedFog&Lighting.esp'
+  url:
+   - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/46289/'
+     name: 'Improved Fog and Lighting'
+  group: *lightingEnhancerGroup
+
+- name: 'ImprovedHearthfireLighting.esp'
```

The indentation needs to be corrected:

```suggestion
  - name: 'ImprovedFog&Lighting.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/46289/'
        name: 'Improved Fog and Lighting'
    group: *lightingEnhancerGroup

  - name: 'ImprovedHearthfireLighting.esp'
```